manim/__main__.py

```diff
@@ -124,8 +124,11 @@ def get_scene_classes_from_module(module):
 def get_module(file_name):
     if file_name == "-":
         module = types.ModuleType("input_scenes")
-        logger.info("Enter the code for your animation and end with an EOF (usually CTRL+D):")
+        logger.info("Enter the animation's code & end with an EOF (CTRL+D or CTRL+Z on Windows):")
```

Comment:
Maybe this is just nitpicking, but I think we should be a bit more specific about the EOF shortcuts, i.e. `CTRL+D on Linux/Unix and CTRL+Z on Windows`

I'll approve this anyways, though

Reply:
> Maybe this is just nitpicking, but I think we should be a bit more specific about the EOF shortcuts, i.e. `CTRL+D on Linux/Unix and CTRL+Z on Windows`
> 
> I'll approve this anyways, though

Yeah, you're right. I wanted to keep the line as short as possible, but clarity is more important.